src/components/common/verify-code/verify-code-controller.ts

```diff
@@ -104,6 +104,9 @@ export function verifyCodePost(
         if (accountInterventionsResponse.data.temporarilySuspended) {
           nextEvent = USER_JOURNEY_EVENTS.TEMPORARILY_BLOCKED_INTERVENTION;
         }
+        if (accountInterventionsResponse.data.passwordResetRequired) {
```

Comment:
I don't follow why we need this? Is this covering off a different bug to the password reset one we were talking about yesterday? Could we have a test for this?

Reply:
Same as above, sorry was entirely misinterpreting this as being the work we talked about yesterday. Test would be good